base/strings/basic.jl

```diff
@@ -227,6 +227,7 @@ Vector{T}(s::AbstractString) where {T<:AbstractChar} = collect(T, s)
 
 Symbol(s::AbstractString) = Symbol(String(s))
 Symbol(x...) = Symbol(string(x...))
+Symbol(x::Symbol, y::Symbol) = Symbol(string(x), string(y)) # 2-arg form improves performance
```

Comment:
```suggestion
Symbol(x::Symbol, y::Symbol) = Symbol(String(x), String(y)) # 2-arg form improves performance
```
For `Symbol`s, `string` just calls `String`, but this makes it a bit more obvious IMO. Feel free to take or leave.